operator/controllers/kyma_controller.go

```diff
@@ -428,6 +388,20 @@ func initModuleFromServer(module *parsed.Module) *unstructured.Unstructured {
 	return &unstructuredFromServer
 }
 
+func (r *KymaReconciler) SyncModuleStatus(ctx context.Context, modules parsed.Modules) {
```

Comment:
no, I don't think so, status helper is designed update `kyma.status` since it has a `StatusWriter`, but this is module status. But after revisit the controller, the module related code is too much, I'm trying to extract them to another pkg.